### ISSUE TITLE
CRLF after `//` comments seem to be converted to LF

### ISSUE THREAD

**Issue by jeddy3** [postcss/postcss-scss#26]
Given a source file *with CRLF line endings*:

```scss
// comment
a {}
```

Then:

```js
JSON.stringify(rule.raw("before")) // gives "\n\r\n" rather than "\r\n\r\n
```
Is this the intended behaviour?

Ref: https://github.com/stylelint/stylelint/issues/845

**Comment by ai** [comment]
Nope, it is a bug. I will look closer tomorrow.

**Comment by ai** [comment]
Done d2c207962eb8462264179a09cbf32a831ab140ca

**Comment by ai** [comment]
Released in 0.1.7.

**Comment by jeddy3** [comment]
Excellent. Thanks for fixing this :)
